sdk/servicebus/service-bus/test/public/sendAndSchedule.spec.ts

```diff
@@ -179,6 +179,29 @@ describe("Sender Tests", () => {
     await testSimpleSendArray();
   });
 
+  it(noSessionTestClientType + "should reject with proper error instead of OperationTimeoutError", async function (): Promise<void> {
+    await beforeEachTest(noSessionTestClientType);
+    const content = (new Array(256 * 1024)).join("x"); //Generate a Large Message Content of 265KB
```

Comment:
fill instead of join?

Reply:
I will keep `fill()` as it's from the original repo in the reported issue and returns a string.